packages/web/src/components/nav/desktop/NavPopupMenu.module.css

```diff
@@ -60,6 +64,12 @@
   height: 100%;
 }
 
+.payAndEarnIcon svg {
+  width: var(--unit-5);
+  height: var(--unit-5) !important;
```

Comment:
why is this `!important`?

Reply:
there is an annoying height property that overrides it otherwise